connectors/Telephony-Xtended-Service/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2445 @@
+{

```

Comment:
@alainmenag Please delete below unwanted files from the commit

.DS_Store
connectors/.DS_Store
connectors/Telephony-Xtended-Service/.DS_Store

Reply:
Removed.